fastlane/Fastfile

```diff
@@ -602,6 +602,7 @@ platform :ios do
 
   desc "Run BackendIntegrationTests"
   lane :backend_integration_tests do |options|
+    fetch_snapshots
```

Comment:
Hmm is this needed? The snapshot tests for paywalls should be independent of these test I thought?

Reply:
The CI tests keep failing on UI Snapshots tests are missing in the backend tests so this was my test to see if this fixed it 🤷‍♂️ Will remove it stops the flakey